service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProvider.java

```diff
@@ -21,11 +21,13 @@
 import static org.apache.commons.lang.StringUtils.capitalize;
 import static org.apache.commons.lang.StringUtils.uncapitalize;
 import static org.apache.commons.lang3.ObjectUtils.defaultIfNull;
+import static uk.gov.hmcts.reform.fpl.service.DateFormatterService.formatLocalDateToString;
 import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.formatCaseUrl;
 import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.getFirstRespondentLastName;
 
+//TODO: refactor out dateFormatterService to provide static methods. Too many knock on effects to do within 1484
+// (FPLA-1482)
 public abstract class AbstractEmailContentProvider {
-
     final String uiBaseUrl;
     private final DateFormatterService dateFormatterService;
```

Comment:
looks like it's not used anymore

Reply:
cleaned up in next PR, I don't want to refactor data formatter service too much in this